src/rastervision/contrib/cowc/transfer_georeference.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
+import os
```

Comment:
I am running in the container, as well.  I am not sure how a version mis-match could occur.

Reply:
`image_file` does not work if there is more than one label raster for the scene. I'm guessing that case won't come up very frequently, but it's possible. For this dataset, it's not a problem.
Re: your second question, I think it's possible for there to be a single label raster, but multiple image rasters per scene.

Reply:
Since this is going to be a "getting started" example, we should probably use `image_file` and avoid having to run the georeferencing script, although it could be useful for another application. 

Here's what I got when I ran the script in the container after running `update`:
root@e820085104e2:/opt/src# ./rastervision/contrib/cowc/transfer_georeference.py \
>     /opt/data/raw-data/isprs-potsdam/4_Ortho_RGBIR/top_potsdam_2_10_RGBIR.tif \
>     /opt/data/raw-data/isprs-potsdam/5_Labels_for_participants_no_Boundary/top_potsdam_2_10_label_noBoundary.tif \
>     /opt/data/raw-data/isprs-potsdam/labels/2_10.tif
Traceback (most recent call last):
  File "./rastervision/contrib/cowc/transfer_georeference.py", line 27, in <module>
    ul = re.search(ul_re, ullr)
  File "/usr/lib/python3.5/re.py", line 173, in search
    return _compile(pattern, flags).search(string)
TypeError: cannot use a string pattern on a bytes-like object